libs/types/src/tokens.rs

```diff
@@ -14,23 +14,26 @@ use core::marker::PhantomData;
 
 use cfg_primitives::{
 	types::{PoolId, TrancheId},
-	Balance, PalletIndex,
+	Balance,
 };
 use cfg_traits::{investments::TrancheCurrency as TrancheCurrencyT, HasLocalAssetRepresentation};
-pub use orml_asset_registry::AssetMetadata;
+use orml_traits::asset_registry;
 use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
-#[cfg(feature = "std")]
 use serde::{Deserialize, Serialize};
 use sp_runtime::{traits::Get, DispatchError, TokenError};
-use xcm::{
-	prelude::{AccountKey20, GlobalConsensus, PalletInstance},
-	v3::{MultiLocation, NetworkId},
-	VersionedMultiLocation,
-};
 
 use crate::{domain_address::DomainAddress, xcm::XcmMetadata, EVMChainId};
 
+pub const MAX_ASSET_STRING_LIMIT: u32 = 128;
```

Comment:
I initially put 64, but I've increased it to be able to fit the previous limit of `128` used for TokenName in `pool-system`. For us, this limit is not an issue because only admin/root can register currencies.